src/constant/enums.js

```diff
@@ -200,3 +234,73 @@ export const PaymentTypeLabels = [
 export function getPaymentTypeLabel(paymentType) {
     return getEnumLabel(PaymentTypeLabels, paymentType);
 }
+
+export const ParkingSize = Object.freeze({
+    Bike: 0,
+    Compact: 1,
+    FullSize: 2,
+    Hatchback: 4,
+    Unspecified: 5,
+});
+
+export const ParkingSizeLabels = [
+    'Bike',
+    'Compact',
+    'FullSize',
+    'Hatchback',
+    'Unspecified',
+];
+
+/**
+ *
+ * @param {int} parkingSize
+ * @return {string}: label for parking size
+ */
+export function getParkingSizeLabel(parkingSize) {
+    return getEnumLabel(ParkingSizeLabels, parkingSize);
+}
+
+
+export const RentUnit = Object.freeze({
```

Comment:
PerMonth Is missing.

https://github.com/parkspot-dev/maya/blob/master/core/models/struct.go#L184